socorro/external/postgresql/models.py

```diff
@@ -944,12 +911,6 @@ class ReleaseChannel(DeclarativeBase):
         secondary='ProductReleaseChannel',
         secondaryjoin='ProductReleaseChannel.product_name==Product.product_name'
     )
-    signatures = relationship(
-        'Signature',
-        primaryjoin='ReleaseChannel.release_channel==Tcbs.release_channel',
-        secondary='Tcbs',
-        secondaryjoin='Tcbs.signature_id==Signature.signature_id'
-    )
```

Comment:
These models are set up as if we used sqlalchemy as an ORM in the backend, but we don't. Everything (and I mean *everything*) is done with raw sql. Further, you can't even use the ORM if you tried because of a bunch of secondary joins that aren't valid.

Given that, I'm just taking the license to remove this because it can't be used in the current condition of things.

Same for the other relationship that was removed in this PR.